### PR TITLE
adds a great dad joke

### PR DESCRIPTION
**Pull request by adebruyne**
This is best joke you have ever read.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,21 @@
+<!-- What do you call a fake noodle? An impasta -->
```
Comment (radishmouse) [inline index.html:1]: so funny i pooped myself

Comment (radishmouse) [reply index.html:1]: i need an adult diaper

Comment (adebruyne) [reply index.html:1]: Heres another! How does a penguin build it's house? Igloos it together

Comment (brienmizell) [reply index.html]: image

**Pull request merged**